opencodelists/tests/functional/test_build_codelist.py

```diff
@@ -118,6 +118,15 @@ def apply(self, page):
             search_action.handle_concept_selections(page)
 
 
+class CodelistHeading(StrEnum):
+    """Represents the heading for a codelist on the My Codelists page."""
+
+    USER_OWNED = "Your codelists"
```

Comment:
This will change after rebasing to main because my recent work on that page has changed 2 headings. "Your codelists" is now "Your published codelists", and "Your drafts" is now "Your draft codelists"